README.md

```diff
@@ -1,6 +1,6 @@
 <h1 align="center">BPMN Visualization</h1>
 <p align="center">
```

Comment:
:warning: The image is not displayed in the html docs (this is not introduced by this PR, but in #643)
The image is not copied when generating the html docs

Reply:
Fixed